lib/ansible/modules/system/java_cert.py

```diff
@@ -13,7 +13,7 @@
 DOCUMENTATION = '''
 ---
 module: java_cert
-version_added: '2.3'
+version_added: '2.4'
```

Comment:
This shoudn't have changed

```suggestion
version_added: '2.3'
```